packages/puppeteer-core/src/common/bidi/BidiOverCDP.ts

```diff
@@ -53,7 +50,7 @@ export async function connectBidiOverCDP(
     // Forwards a BiDi event sent by BidiServer to Puppeteer.
     pptrTransport.onmessage(JSON.stringify(message));
   });
-  const pptrBiDiConnection = new BidiPPtrConnection(cdp.url(), pptrTransport);
+  const pptrBiDiConnection = new BidiConnection(cdp.url(), pptrTransport);
```

Comment:
Not sure of the renaming to `BidiConnection` will have to look what the class does.